packages/components/src/Link/styles.ts

```diff
@@ -0,0 +1,46 @@
+import { Link } from '@mui/material';
+import { theme } from '@astral/tests/src/theme';
+
+import { styled } from '../styles';
+
+export const StyledLink = styled(Link)`
+  padding: ${theme.spacing(0.5, 0.25, 0.5, 0)};
+
+  font-size: ${theme.typography.fontSize}px;
+  color: ${theme.palette.components.link.main};
+
+  &:active {
+    color: ${theme.palette.components.link.active};
+  }
+
+  &:visited {
+    color: ${theme.palette.components.link.visited};
+  }
+
+  &:hover {
+    color: ${theme.palette.components.link.hover};
+
+    * {
+      text-decoration: underline;
+    }
+  }
+
+  &:focus {
+    border-radius: 4px;
+    outline: 2px solid ${theme.palette.primary[400]};
+  }
+
+  & div {
+    display: inline-flex;
+    gap: ${theme.spacing(1)};
```

Comment:
```suggestion
    gap: ${({ theme }) => theme.spacing(1)};
```

С темой в styled компонентах работаем так